aea/aea.py

```diff
@@ -75,13 +74,15 @@ def __init__(
         skill_exception_policy: ExceptionPolicyEnum = ExceptionPolicyEnum.propagate,
         loop_mode: Optional[str] = None,
         runtime_mode: Optional[str] = None,
+        default_connection: Optional[PublicId] = None,
```

Comment:
Should this be `PublicId = DEFAULT_CONNECTION_ID` where the latter is the StubConnection id?